docs/rules/no-v-html.md

```diff
@@ -32,12 +32,42 @@ This rule reports all uses of `v-html` directive in order to reduce the risk of
 
 ## :wrench: Options
 
-Nothing.
+```json
+{
+    "vue/no-v-html": ["error", {
+        "ignorePattern": ".Html$"
```

Comment:
I think the `.` is not necessary, since the pattern is not matched against the whole string. Please remove it everywhere in the docs and tests.

Reply:
> How about we make the examples match a _prefix_ `html` instead, i.e. `^html`?

Sounds good to me :slightly_smiling_face:

Reply:
Ok, done :)